src/cross_module_fn.c

```diff
@@ -22,6 +23,7 @@
 	}
 
 /* bgw policy functions */
+CROSSMODULE_WRAPPER(is_compressed_tid);
 CROSSMODULE_WRAPPER(policy_compression_add);
```

Comment:
`is_compressed_tid` is not a bgw policy function, right?

Reply:
Nope, it is a function to check the TID of the tuple and decide if it is a compressed or non-compressed tuple.

Reply:
We should move the comment.

Reply:
Done. Added a new section with functions for this.